src/env/mod.rs

```diff
@@ -7,14 +7,25 @@ use std::sync::Arc;
 mod default;
 mod obfuscated;
 
+pub use default::AioContext;
```

Comment:
remove this, as explained in https://github.com/tikv/raft-engine/pull/286/files#r1058018164